src/olympia/accounts/tests/test_views.py

```diff
@@ -651,11 +649,51 @@ def test_username_taken_creates_user(self):
         assert not user.has_usable_password()
 
 
+@override_settings(FXA_CONFIG={
+    'foo': {'FOO': 123},
+    'bar': {'BAR': 456},
+    'baz': {'BAZ': 789},
+})
+class TestFxAConfigMixin(TestCase):
+
+    class DefaultConfig(views.FxAConfigMixin):
+        DEFAULT_FXA_CONFIG_NAME = 'bar'
+
+    class MultipleConfigs(views.FxAConfigMixin):
+        DEFAULT_FXA_CONFIG_NAME = 'baz'
+        ALLOWED_FXA_CONFIGS = ['foo', 'baz']
+
+    def test_default_only_no_config(self):
+        request = RequestFactory().get('/login')
+        config = self.DefaultConfig().get_fxa_config(request)
+        assert config == {'BAR': 456}
+
+    def test_default_only_not_allowed(self):
+        request = RequestFactory().get('/login?config=foo')
+        config = self.DefaultConfig().get_fxa_config(request)
+        assert config == {'BAR': 456}
+
+    def test_default_only_allowed(self):
+        request = RequestFactory().get('/login?config=bar')
+        config = self.DefaultConfig().get_fxa_config(request)
+        assert config == {'BAR': 456}
+
+    def test_config_is_allowed(self):
+        request = RequestFactory().get('/login?config=foo')
+        config = self.MultipleConfigs().get_fxa_config(request)
+        assert config == {'FOO': 123}
+
+    def test_config_is_default(self):
+        request = RequestFactory().get('/login?config=baz')
+        config = self.MultipleConfigs().get_fxa_config(request)
+        assert config == {'BAZ': 789}
```

Comment:
Is there a test for a query string that tries to get a config that's not allowed? I don't see it.